src/test/java/com/n1analytics/paillier/PaillierEncryptedNumberTest.java

```diff
@@ -1035,11 +1035,11 @@ public void testCalculateCiphertext() throws Exception {
     @Test
     public void testCiphertextObfuscation() throws Exception {
```

Comment:
my interpretation of the goal of this test is that we want to check if a non-obfuscated EncryptedNumber changes after obfuscation. Therefore it would not make much sense to call the `calculateCiphertext()` function here.